cluster-operator/src/test/java/io/strimzi/operator/cluster/model/CruiseControlTest.java

```diff
@@ -143,23 +147,23 @@ public class CruiseControlTest {
             .withImage(ccImage)
             .withConfig(ccConfig)
             .withNewTemplate()
-                .withNewPod()
-                    .withTmpDirSizeLimit("100Mi")
-                .endPod()
+            .withNewPod()
+            .withTmpDirSizeLimit("100Mi")
+            .endPod()
```

Comment:
The indentation makes things more readable ... so maybe you can keep it?

Reply:
Indentation fixed